examples/Madagascar_RTC_autorunonce

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+hostname='ole'
+hotspotname='treehouse'
+port='5984'
+
+# real time clock: rasclock or ds3231
+rtc='rasclock'
+# rtc='ds3231'
+
+# rename hostname
+pirateship rename $hostname
+
+# create couchdb docker container
+docker run -d -p $port:5984 --name bell -v /srv/data/bell:/usr/local/var/lib/couchdb -v /srv/log/bell:/usr/local/var/log/couchdb dogi/rpi-couchdb
+
+# expand filesystem
+cd /usr/local/lib/
+npm update
+pirateship expandfs
+
+# redirect to bell
+mkdir -p /root/ole
+echo '#!/usr/bin/env node' > /root/ole/server.js
+echo '' >> /root/ole/server.js
+echo "var express = require('express')" >> /root/ole/server.js
+echo 'var PortJack = express()' >> /root/ole/server.js
+echo 'PortJack.get(/^(.+)$/, function(req, res) {' >> /root/ole/server.js
+echo 'var options = {' >> /root/ole/server.js
+echo '"'$hostname'.local": "http://'$hostname'.local:5984/apps/_design/bell/MyApp/index.html",' >> /root/ole/server.js
+echo '"192.168.2.1": "http://192.168.2.1:5984/apps/_design/bell/MyApp/index.html",' >> /root/ole/server.js
+echo '"192.168.0.99": "http://192.168.0.99:5984/apps/_design/bell/MyApp/index.html",' >> /root/ole/server.js
+echo '"192.168.1.99": "http://192.168.4.99:5984/apps/_design/bell/MyApp/index.html"' >> /root/ole/server.js
+echo '}' >> /root/ole/server.js
+echo 'if (options.hasOwnProperty(req.hostname)) {' >> /root/ole/server.js
+echo "res.setHeader('Location', options[req.hostname])" >> /root/ole/server.js
+echo '}' >> /root/ole/server.js
+echo 'else {' >> /root/ole/server.js
+echo "res.setHeader('Location', 'http://ole.org')" >> /root/ole/server.js
+echo '}' >> /root/ole/server.js
+echo 'res.statusCode = 302' >> /root/ole/server.js
+echo 'res.end()' >> /root/ole/server.js
+echo '})' >> /root/ole/server.js
+echo 'PortJack.listen(80)' >> /root/ole/server.js
+chmod +x /root/ole/server.js
+cd /root/ole
+npm install express
+
+# real time clock
+cd /boot/overlays
+echo >> /boot/config.txt
+
+if [ "$rtc" = "rasclock" ]; then
+  echo 'dtoverlay=i2c-rtc,pcf2127' >> /boot/config.txt
+elif [ "$rtc" = "ds3231" ]; then
+  echo 'dtoverlay=i2c-rtc,ds3231' >> /boot/config.txt
+fi
+
+cp /lib/udev/hwclock-set /lib/udev/hwclock-set.old
+sed -e '/run\/systemd\/system/,+2 s/^#*/#/' -i /lib/udev/hwclock-set
+sed -e '/--systz/ s/^#*/#/' -i /lib/udev/hwclock-set
+
+# disable fake-hwclock
+sudo apt-get -y remove fake-hwclock
+sudo update-rc.d -f fake-hwclock remove
+
+# second autorunonce
+mv /boot/autoranonce* /boot/autoranonce.first
+sync
+cat <<EOF > /boot/autorunonce
+#!/bin/sh
+sleep 10
+while ! curl -X GET https://google.com ; do
```

Comment:
kinda scared about this one,
if the user doesn't have an ehternet/wifi connected it will never boot?

Reply:
I know but so far have no better solution ...
should we raise that working upstream ethernet is required

Reply:
a weird solution would be to put this on a cron job (or find a way to run this when an internet connection is found if-up.d maybe?).

once the cronjob is done (internet connection found) remove the entry -crontab- from the system

Reply:
hmm maybe a task for an other issue ;)